app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    not_vaccinated = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
```

Comment:
Catching 'VaccineError' will handle both 'NotVaccinatedError' and 'OutdatedVaccineError'. If you want to differentiate between these two cases, catch them separately.